src/main.ts

```diff
@@ -1,8 +1,20 @@
 import { NestFactory } from '@nestjs/core';
+import { SwaggerModule, DocumentBuilder } from '@nestjs/swagger';
 import { AppModule } from './app.module';
 
 async function bootstrap() {
   const app = await NestFactory.create(AppModule);
+
+  const config = new DocumentBuilder()
+    .setTitle('萬隆 Line Bot API')
+    .setDescription('API 說明文件')
+    .setVersion('1.0')
+    .addTag('0.1.0')
```

Comment:
Let's align this version with the one in package.json

Reply:
Wow, it turns out we need to match the version with package.json. 
Thank you for your suggestion. 
I will change the API Document's version to 0.0.1.